deployments/kai-scheduler/templates/binder-webhook.yaml

```diff
@@ -4,7 +4,7 @@
 apiVersion: admissionregistration.k8s.io/v1
 kind: MutatingWebhookConfiguration
 metadata:
-  name: mutating-binder
+  name: kai-binder-mutating
```

Comment:
Why do we need this suffix?
When you have cluster role of scheduler you dont call it "kai-scheduler-clusterrole" because its already type of clusterrole.
So in here the type is MutatingWebhookConfiguration, we do we need the mutating at the end?

Reply:
I will remove